scripts/ci/check_compliance.py

```diff
@@ -284,7 +306,7 @@ def parse_kconfig(self):
         os.environ["ARCH_DIR"] = "arch/"
         os.environ["BOARD_DIR"] = "boards/*/*"
         os.environ["ARCH"] = "*"
-        os.environ["CMAKE_BINARY_DIR"] = tempfile.gettempdir()
+        os.environ["KCONFIG_BINARY_DIR"] = tempfile.gettempdir()
```

Comment:
I guess this is using /tmp too, so maybe it's fine...

Reply:
yeah, it is existing behavior, but doesn't mean it is safe.
See other comment.